README.md

```diff
@@ -39,6 +39,16 @@ Comments are enabled by default and will only appear in production, i.e., `JEKYL
 
 If you don't want to display comments for a particular post you can disable them by adding `comments: false` to that post's YAML Front Matter.
 
+## Enabling Google Analytics
+
+To enable GA, add the following lines to your Jekyll site:
```

Comment:
I'd spell out Google Analytics each time. Space is cheap and it can help new users understand what's going on.